### DISCUSSION TITLE
How to enable sort: requiredFirst for Controls blocks in MDX?

### DISCUSSION THREAD

**Discussion by genepaul** [storybookjs/storybook#23553] [Help]
I've looked through the documentation, and it seems like I should be able to do this by adding `sort: 'requiredFirst'` to a story's parameters:

https://storybook.js.org/docs/7.0/react/essentials/controls?_sm_au_=iHV4tJjfksn3DH3HL321jK0f1JH33#sorting-controls

However, this doesn't seem to affect when I am using MDX to display my stories. The only way I can seem to get a `<Controls/>` block in MDX to sort its props required first is to pass it as a prop to the block in MDX: `<Controls sort={'requiredFirst'}/>`. Is this just not possible? Ideally, I'd like to add this as a global config in `preview` and have it affect all my Controls blocks. This seems to work for ArgsTable, but not Controls. Am I using the wrong block?

**Comment by urrri-redis** [comment]
use 

```
const preview = {
    controls: {
      sort: 'requiredFirst' // for stories
    }
    docs: {
      controls: {
        sort: 'requiredFirst' // for docs
      }
    },

}
```
